ImperatorToCK3/Data_Files/configurables/culture_map.txt

```diff
@@ -0,0 +1,8 @@
+# Usage:
+# link = { ck3 = ck3Culture imp = impCulture province = ck3provinceID owner = ck3tag }
```

Comment:
I probably will use d/k/e instead of regions, indeed. Will open an issue for this and add the support in a future PR, since this one is already pretty big.